packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1084,6 +1085,10 @@ export function catalogServiceBuilder(
         assertRiskAnalysisIsValidForPublication(eservice.data, tenant.kind);
       }
 
+      if (descriptor.audience.length === 0) {
+        throw audienceCannotBeEmpty(descriptor.id);
+      }
+
```

Comment:
@galales does this need to be also in `activateDescriptor` (publication after suspension)? Theoretically, if we just check this here and don't allow modifications to this field after the publication, there should be no need.

Reply:
I agree, it's enough to check it here until audience cannot be changed